lib/symbioticpy/symbiotic/targets/tool.py

```diff
@@ -164,7 +164,7 @@ def slicer_options(self):
         if prop.signedoverflow():
             # default config file is 'config.json'
             # slice wrt asserts checking for the overflow
-            return (['__assert_fail'],[])
+            return (['__assert_fail', '__VERIFIER_error'],[])
```

Comment:
Now we want just `__VERIFIER_error` here, right?

Reply:
You're right, we don't use asserts anymore. I'll fix it right away.